### ISSUE TITLE
No focus feedback during keyboard navigation for emoji picker and smart picker

### ISSUE THREAD

**Issue by nimishavijay** [nextcloud/mail#9463]
### Steps to reproduce

1. Open the composer
2. Type `/` or `:` in the body to open the smart picker or emoji picker
3. Press the down arrow 

### Expected behavior

a background should appear behind the focused item in the smart picker/emoji list

![image](https://github.com/nextcloud/mail/assets/52440189/7d52e207-2000-4722-a324-6500822c9fbf)

### Actual behavior

the item is selected (tab or enter inserts the emoji/opens the smart picker dialog) but there is no visual feedback 

![image](https://github.com/nextcloud/mail/assets/52440189/4acb7e5c-55ea-4345-aa74-5b0886c4ad84)

### PHP engine version

None

### Web server

None

### Database

None
